src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import './UserInfo.scss';
+import { User } from '../../types/userType';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => user && (
```

Comment:
when you describe the Props interface you wrote that a user is required.
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => (
```
<img width="225" alt="Screenshot 2022-09-13 at 20 16 11" src="https://user-images.githubusercontent.com/47606379/189965405-ca11db81-253d-413d-ac38-ec3cc594e4ed.png">